Os/Posix/CMakeLists.txt

```diff
@@ -7,26 +7,62 @@
 ####
 restrict_platforms(Posix)
 add_custom_target("${FPRIME_CURRENT_MODULE}")
+
```

Comment:
When I run `fprime-util check` here, I see failed tests and errors reporting that executable files do not exist. Perhaps there are missing build dependencies? The missing files seem to be
```
build-fprime-automatic-native-ut/bin/Darwin/PosixTaskTest
build-fprime-automatic-native-ut/bin/Darwin/StubTaskTest
```

Reply:
I will take this as a bug.  It is an error in the CMake system w.r.t. using named (but no module) UTs.  As such this bug should be solved outside of this PR.

Reply:
#2734